api/tests/LocalAuthBearerTokenTest.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+use App\Services\LocalAuthBearerTokenService;
+use App\Services\Contracts\BearerTokenServiceInterface;
+
+class LocalAuthBearerTokenTest extends TestCase
+{
+    /**
+     * @var BearerTokenServiceInterface
+     */
+    protected $service_provider;
+
+    protected function setUp(): void
+    {
+        // generate keys and tokens for testing at https://jwt.io/#debugger-io
+        // make sure you set algorithm to RS256
+        $issuer = 'http://test.com';
+        $publicKey = file_get_contents('tests/resources/key1-cert.pem');
+        if(!$publicKey)
+            throw new Exception('Failed to load public key from file');
+        $timeZone = 'UTC';
+
+        $this->service_provider = new LocalAuthBearerTokenService($issuer, $publicKey, $timeZone);
+
+        parent::setUp();
+
+    }
+    /**
+     * @test
+     * A valid token is provided and validated.  The test checks that the right sub value is returned.
+     */
+    public function it_accepts_a_good_token_and_returns_the_right_sub()
+    {
+        //$token = 'eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJodHRwOi8vdGVzdC5jb20iLCJzdWIiOiIxMjM0NTY3ODkwIiwiaWF0IjowLCJleHAiOjIxNDc0ODM2NDd9.G8HuIKwidOxRTgTKQCw1Lo_rU0j8PD57i9TfY5RIYqo3osCILwyDKCd31f6luZvy49PT4R_x-_ZOFMmkb6_pnuJGSHYv_EpZcYWCrI2X5USox4IrQ4mmFGZP8pPJOeBIAS00i_Unab-LR2mnqNCT-EsZOA7sHcJTDyNB4FSsHtmZNKXtsw335DSuDb6nIa77SV1gmhPIA7LKnxgSvacxtSDr9LbEqX-mwXfMSxTOWsLosBwqHHjc2W3tWlPHEAkLqGMsEl6miZK7RSAPXUE16vlGM9QccCidZWXBXi9lZ_tVkHIIsRIXdiS1RTc8VMtumi1_vRMFU4dheypRQQ68ww';

```

Comment:
Can this be removed?

Reply:
Good catch.  This was an old token that I forgot to remove.